docs/product/session-replay/replay-details.mdx

```diff
@@ -51,3 +51,14 @@ Find a complete list of built-in fields and custom tags associated with a replay
 ## Retention
 
 Replays are retained for 90 days in sentry.io (the same retention period as other event types, such as Errors and Transactions). The retention period is not configurable.
+
+## Inbound Filtering
+
+Session replay supports some inbound filtering rules.  Those rules are:
```

Comment:
```suggestion
Session Replay supports some inbound filtering rules.  Those rules are:
```

Reply:
Is this the correct phrasing? It's not that inbound filters apply to Session Replay specifically. It's that **errors** that are dropped due to the aforementioned Inbound Filters, won't also be captured as part of`replaysOnErrorSampleRate` for Session Replay. I think that nuance is important